### PR TITLE
chore: update zeromq to 0.4.1

### PR DESCRIPTION
**Pull request by uncomfyhalomacro**
This PR ~~also regenerates Cargo.lock~~ comes with the new `zeromq` version.

Closes: #26810

### CODE REVIEW

File: Cargo.lock
Comment (bartlomieju) [inline Cargo.lock:1]: We should revert this whole file and only apply changes related to `zeromq`

Comment (uncomfyhalomacro) [reply Cargo.lock:1]: will do

Comment (uncomfyhalomacro) [reply Cargo.lock:1]: now resolved in latest commit

**Review by bartlomieju — APPROVED** [review]
Thanks

**Pull request merged**